### DISCUSSION TITLE
Postal SMTP logs

### DISCUSSION THREAD

**Discussion by uhlhosting** [postalserver/postal#1945] [Help with using Postal]
Hi,

It seems I cannot get SMTP logs properly, all we got is this;

```
smtp_1      | /usr/local/bundle/gems/gelf-3.0.0/lib/gelf/notifier.rb:65: warning: constant ::Fixnum is deprecated
smtp_1      | log writing failed. "\xFC" from ASCII-8BIT to UTF-8
smtp_1      | log writing failed. "\xAF" from ASCII-8BIT to UTF-8
smtp_1      | log writing failed. "\xFC" from ASCII-8BIT to UTF-8
smtp_1      | log writing failed. "\xAC" from ASCII-8BIT to UTF-8
smtp_1      | log writing failed. "\xFC" from ASCII-8BIT to UTF-8
smtp_1      | log writing failed. "\xC0" from ASCII-8BIT to UTF-8
smtp_1      | log writing failed. "\xFC" from ASCII-8BIT to UTF-8
smtp_1      | log writing failed. "\x88" from ASCII-8BIT to UTF-8
smtp_1      | log writing failed. "\x9B" from ASCII-8BIT to UTF-8
```

**Comment by uhlhosting** [comment]
Had to turn it on from settings, since it seems is not on by default. Yet the logs are the same, if anyone has some resolution to this?

**Reply by willpower232** [reply]
which setting did you change sorry?

**Reply by uhlhosting** [reply]
![image](https://user-images.githubusercontent.com/6324047/163007949-b12800da-9cb3-4e1c-8737-2d5df2c68d3c.jpeg)

**Reply by willpower232** [reply]
That setting means that it will log the contents of messages which normally wouldn't happen as it would bloat the logfiles if left enabled for too long.

There are a few references to this error message on search engines so it seems to be a ruby thing but I wonder if maybe it is having a bad time with the name of your server or something if it contains a letter with an accent or otherwise outside of the boring a-z?

**Reply by uhlhosting** [reply]
my server is post.uhlhosting.ch nothing fancy on it.

**Reply by uhlhosting** [reply]
@willpower232 any possible fix in this matters? Up to this day I cannot use CLI like: `postal logs smtp` because they will be all non sense like above.
